cmd/buildkitd/config/config.go

```diff
@@ -64,6 +64,9 @@ type OCIConfig struct {
 	Rootless         bool              `toml:"rootless"`
 	NoProcessSandbox bool              `toml:"noProcessSandbox"`
 	GCConfig
+	// UserRemapUnsupported is unsupported key for testing. The feature is
+	// incomplete and the intention is to make it default without config.
```

Comment:
ready to merge?

Reply:
Yes, we'll rename to `UserRemap` when the feature is complete

Reply:
or remove completely